src/components/main/ImportH5P.js

```diff
@@ -63,33 +65,27 @@ const ImportH5P = () => {
   return (
     <>
       <Typography variant="h6">{t('Import H5P rich content')}</Typography>
-      <Typography variant="body">
+      <Typography variant="body" paragraph>
         {t(
           'You can upload H5P rich content by uploading exported .h5p files (e.g. from H5P.com, external Moodle services, etc).',
         )}
       </Typography>
-      <br />
-      <Typography variant="body">
+      <Typography variant="body" paragraph>
         {t(
           'Once your file is accepted, it will take several minutes for it to be available.',
         )}
       </Typography>
+      <Typography variant="body" paragraph>
+        {t(`You can upload up to one ZIP of SIZE at a time.`, {
+          maxSize: prettyBytes(MAX_ZIP_FILE_SIZE),
+        })}
+      </Typography>
       <div id={H5P_DASHBOARD_UPLOADER_ID}>
         <Dashboard
           uppy={uppy}
           height={200}
           width="100%"
           proudlyDisplayPoweredByUppy={false}
-          note={t('Upload a file')}
```

Comment:
Do we remove the text hints?

Reply:
I've removed them because I couldn't make them show. Anyway we need to upgrade the uppy dependencies so we can move them back later.